testing/ui-tests/Cargo.toml

```diff
@@ -10,8 +10,10 @@ publish = false
 
 [dev-dependencies]
 trybuild = { workspace = true }
+hex = { workspace = true }
 scale-info = { workspace = true, features = ["bit-vec"] }
 frame-metadata ={ workspace = true }
 codec = { package = "parity-scale-codec", workspace = true, features = ["derive", "bit-vec"] }
-subxt = { workspace = true, features = ["native", "jsonrpsee"] }
+subxt = { workspace = true, features = ["native", "jsonrpsee", "substrate-compat"] }
```

Comment:
Where is `substrate-compat` needed? Offhand I can't see any use for it but I'm likely to be missing something :)